index.html

```diff
@@ -13,30 +14,29 @@
             <img src="#" alt="main-logo" class="main-logo" />
           </a>
         </li>
+        <li class="nav-item">
```

Comment:
logo its not a list item 

A list is an enumeration of homogeneous elements with the same meaning and styling. This means that any number of items can be added to the list, and they should all look the same. A heading stands out from this enumeration and is not part of the list.

Reply:
and you duplicate logo in this list